AnnoDesigner/Localization/TranslateExtension.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace AnnoDesigner.Localization
+{
+    public class TranslateKeyConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var translations = value as IDictionary<string, string>;
+            var key = parameter as string;
+
+            if (translations.ContainsKey(key))
```

Comment:
We can use `TryGetValue()` here instead.

Reply:
I'll combine the this issue with one above to rewrite this to `if (parameter is string key && translation.TryGetValue(key, out translated))`